README.md

```diff
@@ -1,3 +1,247 @@
-# swift-quickstart-platform
+# PubNub Swift Sample App
 
 This repository contains the files required to run the [PubNub Swift Quickstart](https://www.pubnub.com/docs/platform/quickstarts/swift).
+
+PubNub takes care of the infrastructure and APIs needed for the realtime communication layer of your application. Work on your app's logic and let PubNub handle sending and receiving data across the world in less than 100ms.
+
+## Get keys
+
+You will need the publish and subscribe keys to authenticate your app. Get your keys from the [Admin Portal](https://dashboard.pubnub.com/login).
+
+## Setup the project
+
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
+
+
+1. Open Xcode and create a new iOS App.
+2. Give it a Product Name, `pnquickstart`.
+3. Set the Language to Swift.
+4. Set User Interface to SwiftUI.
+5. Set Lifecycle to UIKitAppDelegate.
+6. Save the project to your file system.
+7. Navigate to File > Swift Packages > Add Package Dependency, enter the package repository URL: https://github.com/pubnub/swift.git and click the dialog's Next and Finish buttons, keeping all default options.
+
+## Add project files
+
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
```

Comment:
Duplicate line

Reply:
We're duplicating this one on purpose, in all the READMEs. Good reminder given we're about to ask you to copy-paste almost 200 lines. :)